### PR TITLE
MP Client: Add cl_coloredTextShadows cvar

### PR DESCRIPTION
**Pull request by AlexCSilva**
This cvar when toggled on will enable the coloredTextShadows feature from Jedi Outcast. Set to 0 by default.

### CODE REVIEW

**Comment by ensiform** [comment]
Quite certain the bug preventing them was already fixed.

**Comment by AlexCSilva** [comment]
Really?

**Comment by AlexCSilva** [comment]
It still bit buggy in chat sometimes so people may want it turned off. Example of this is if u type ^ 7 and a duplicate 7 would show inside the ^ on the chat. lol

**Comment by ensiform** [comment]
Check the history of the font file you modified, I'm not at PC right now to look easily.

**Comment by ensiform** [comment]
You'll have to explain with images then.

**Comment by AlexCSilva** [comment]
https://puu.sh/w0qAs/2e531de09b.jpg

**Comment by AlexCSilva** [comment]
i should have write Client MP lol oops

**Comment by AlexCSilva** [comment]
Is the unsuccessful check just to do with the compile or is it also code issue? just wanderin

**Comment by xycaleth** [comment]
Looks like a problem with the build system. I've restarted the compile job so hopefully it works now.

**Comment by AlexCSilva** [comment]
for example you put ^5^5^7Name Here then you'll get a name that is white but with cyan shadow in background.

File: codemp/rd-common/tr_font.cpp [hunk 1]
```diff
@@ -1548,15 +1548,43 @@ void RE_Font_DrawString(int ox, int oy, const char *psText, const float *rgba, c
...
+				//^blah stuff confuses shadows, so parse it out first
+				while (psText[i] && r < 1024) {
+					if (psText[i] == '^') {
+						if ((i < 1 || psText[i - 1] != '^') &&
+							(!psText[i + 1] || psText[i + 1] != '^')) { //If char before or after ^ is ^ then it prints ^ instead of accepting a colorcode
+							i += 2;
+						}
+					}
 
-		const vec4_t v4DKGREY2 = {0.15f, 0.15f, 0.15f, rgba?rgba[3]:1.0f};
```
Comment (AlexCSilva) [inline codemp/rd-common/tr_font.cpp]: oic i should fix that

**Comment by AlexCSilva** [comment]
broke this pull request lmfao

**Pull request closed without merging**